docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -50,6 +50,24 @@ If you need more predictable behavior across releases, reference the particular
 
 = User Profile Changes
 
+== User profile enabled by default
+
+User profile feature is now enabled by default. The feature `declarative-user-profile` is now not available anymore as the user profile is always considered enabled now.
```

Comment:
```suggestion
The user profile feature is now enabled by default. The  `declarative-user-profile`feature is no longer available, because the user profile is assumed to be always enabled.
```